buildenv

```diff
@@ -1,6 +1,9 @@
-export ZOPEN_GIT_URL="https://github.com/junegunn/fzf.git"
-export ZOPEN_GIT_DEPS="make"
-export ZOPEN_TYPE="GIT"
+# bump: fzf-version /FZF_VERSION="(.*)"/ https://github.com/junegunn/fzf.git|semver:*
+FZF_VERSION=""
```

Comment:
This would probably fail the zopen_get_version check. We probably need to add a STABLE line and point to a tagged release